packages/xo-web/src/xo-app/sr/tab-xostor.js

```diff
@@ -0,0 +1,104 @@
+import _ from 'intl'
+import Component from 'base-component'
+import Icon from 'icon'
+import React from 'react'
+import SortedTable from 'sorted-table'
+
+import { addSubscriptions, connectStore } from 'utils'
+import { Card, CardHeader, CardBlock } from 'card'
+import { Container, Row, Col } from 'grid'
+import { createSelector, createGetObjectsOfType } from 'selectors'
+import { Host, Vdi } from 'render-xo-item'
+import { subscribeXostorHealthCheck } from 'xo'
+
+const RESOURCE_COLUMNS = [
+  {
+    name: 'Resource name',
+    itemRenderer: ({ resourceName }) => resourceName,
+    sortCriteria: ({ resourceName }) => resourceName,
+  },
+  {
+    name: _('node'),
+    itemRenderer: ({ host }) => <Host id={host.id} link />,
+    sortCriteria: ({ host }) => host.name_label,
+  },
+  {
+    name: _('nodeStatus'),
+    itemRenderer: ({ nodeStatus }) => nodeStatus,
+    sortCriteria: ({ nodeStatus }) => nodeStatus,
+  },
+  {
+    name: _('vdi'),
+    itemRenderer: ({ vdiId }) => <Vdi id={vdiId} />,
+  },
+  {
+    name: _('inUse'),
+    itemRenderer: resource => <Icon icon={String(resource['in-use'])} />,
```

Comment:
What are the possible values of `resource['in-use']`? I don't see that property when I test it.

Reply:
It return a boolean.
MB, the property has moved after the last Ronan update on the `health-check`. I will fix